prog/vm/nexus.rb

```diff
@@ -193,7 +194,7 @@ def start
 
     vm_host_id = allocate
     vm_host = VmHost[vm_host_id]
-    ip4, address = vm_host.ip4_random_vm_network
+    ip4, address = vm_host.ip4_random_vm_network if vm.ip4_enabled
```

Comment:
If ip4_enabled but vm_host doesn't have enough ipv4 block, it should fail. But I think it will continue without ipv4 currently